src/app/[locale]/(default)/about/page.tsx

```diff
@@ -1,22 +1,116 @@
-import { getTranslations, unstable_setRequestLocale } from 'next-intl/server';
-
-export async function generateMetadata({
-  params: { locale },
-}: {
-  params: { locale: string };
-}) {
-  const t = await getTranslations({ locale, namespace: 'layout' });
-
-  return {
-    title: t('about'),
-  };
-}
+import { unstable_setRequestLocale } from 'next-intl/server';
+import { Button } from 'src/components/ui/Button';
+import React from 'react';
+import { Card, CardContent, CardHeader } from 'src/components/ui/Card';
+import { useTranslations } from 'next-intl';
+import { Map, Printer, Gamepad2, SquareUserRound } from 'lucide-react';
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from '@/components/ui/Accordion';
+import { Meteors } from '@/components/ui/Meteor';
+import Image from 'next/image';
+import Link from 'next/link';
 
 export default function AboutPage({
   params: { locale },
 }: {
   params: { locale: string };
 }) {
   unstable_setRequestLocale(locale);
-  return <div>this should be about page</div>;
-}
+  const t = useTranslations('about');
+
+const cardData = [
```

Comment:
you dont need translated content for mock objects. translations will come from the database and not be handled on the web server